src/CodeMirrorBlocks.js

```diff
@@ -19,8 +20,21 @@ export default class CodeMirrorBlocks {
       />,
       container
     );
+    merge(api, this.buildAPI());
     return api;
   }
+
+  buildAPI = () => {
+    return {
+      'fromTextArea': this.fromTextArea,
+    }
+  }
+
+  fromTextArea = myTextArea => {
```

Comment:
This is based on:
```
var myCodeMirror = CodeMirror(function(elt) {
  myTextArea.parentNode.replaceChild(elt, myTextArea);
}, {value: myTextArea.value});
```
from the CM documentation. This isn't the main point of this PR, but I would still love your thoughts on whether y'all think this will work.

Reply:
This looks like the right approach to replace the text area, but `ToggleEditor` needs more props than that. Probably the constructor should save its arguments, and then pass them into `fromTextArea`. And since it will not build `ToggleEditor` in two places, it should probably have a helper function to do that because otherwise it would be easy to forget one prop in one of the two places.

I'm liking this pattern of layers of things calling `merge(api, this.buildAPI())`.

Reply:
I don't have anything to add to what Justin said. Since I'm snowed under working on the talk, any chance you feel like folding the alphabetization into this PR? :)

Reply:
Okay! Let us know how the talk goes!

Reply:
Alphabetized in commit aa3b325